src/components/views/rooms/EventTile.tsx

```diff
@@ -87,7 +87,11 @@ import { isLocalRoom } from '../../../utils/localRoom/isLocalRoom';
 import { ElementCall } from "../../../models/Call";
 import { UnreadNotificationBadge } from './NotificationBadge/UnreadNotificationBadge';
 
-export type GetRelationsForEvent = (eventId: string, relationType: string, eventType: string) => Relations;
+export type GetRelationsForEvent = (
+    eventId: string,
+    relationType: RelationType | string,
+    eventType: EventType | string,
+) => Relations | undefined;
```

Comment:
Not every event has relations so of course we should be able to return `undefined`.

Unless of course there can be an empty `Relations` object? Since `Relations(relationType, eventType, client)` requires `relationType` I don't think this is the case.

Reply:
Re-commented on the diff since this got outdated, https://github.com/matrix-org/matrix-react-sdk/pull/9558#discussion_r1017233677